en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/gateway-policies.md

```diff
@@ -0,0 +1,98 @@
+#Gateway Policies
+
+In WSO2 API Manager, managing Global Level Policies within the gateway infrastructure is made easier. This streamlines policy handling, eliminating the need for administrators to manually create XML-type policy files. This feature facilitates a user-friendly approach to handling policies, eliminates the requirement for manual XML creation and placement in specific directories, and enhances control by providing a clear interface for creating, deploying, and undeploying policies, offering administrators better control over policy management.
+
+!!! note
+    Policies added globally using the older approach: creating XML files following the naming pattern `WSO2AM--Ext--<DIRECTION>` and saving them in the `<API-M_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory, will not be overridden by the global policies added through the UI. Initially, the system will execute the policies added via manual files and then proceed to execute the policies deployed from the UI.
+
+!!! warning
+    A specific gateway with active global policy deployments will apply to all the APIs deployed on that gateway. Similarly, this applies to tenants within the tenant available setup. Global policies will be applied to all APIs belonging to a specific gateway in the tenant corresponding to the logged-in tenant during the deployment of the global policy.
+
+!!!info
+    Although all managing actions can be taken by Admin users with specific scopes, API creators and publishers can view the meta information of global policies such as names, descriptions, policy mappings, and deployed gateway labels.
+
+## Step 1 - Create a gateway policy
+
+Follow the instructions provided below to create a gateway policy:
+
+1. {!includes/sign-in-publisher.md!}
+
+2. Navigate to **Global Policies** in the left menu and proceed to select **Global Policies**.
+
+    The **Create A New Global Policy** page appears.
+
+    [![Global policies startup]({{base_path}}/assets/img/deploy/gateway/global-polices-fresh-start.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-fresh-start.png)
+
+3. Update the policy name and description, then simply drag and drop the desired policies onto the respective flows to attach them.
+
+    Let's create a sample gateway policy mapping following the steps below:
+
+    - Navigate to the Request tab of the Policy List.
```

Comment:
As these are separate steps, let's make them numbered. May be as a), b), c).

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7526/commits/64bafa0aa418ed0dbbc05491694766f9d24f000b.
Thank you for your suggestion.

Screenshot with changes:
![screencapture-localhost-8000-deploy-and-publish-deploy-on-gateway-api-gateway-gateway-policies-2024-01-05-16_08_29](https://github.com/wso2/docs-apim/assets/28702407/10bbfc19-b441-47c8-9192-9cafaa2ee7af)